src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -375,6 +388,17 @@ export type OncokbCancerGene = {
     isCancerGene: boolean;
 };
 
+export type OncokbCancerStructVar = {
```

Comment:
Not available from OncokbAPI?  looping in @zhx828

Reply:
Also, if we must keep, lets put this in models folder

Reply:
I don't think we have Structural Variant class but `AnnotateStructuralVariantQuery`